File: ws/enriquez/unit2/ws23/GeometryCode/Geometry/Circle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Geometry
{
    class Circle : IShape
    {
        public double Radius { get; set; }

        public Circle(double radius)
        {
            Radius = radius;
        }

        public double calculateArea()
        {
            return Math.PI * Math.Pow(Radius, 2);
        }

        public double calculatePerimeter()
        {
            return 2 * Math.PI * Radius;
        }
    }
}

File: ws/ansatuña/u2/ws23OOPrincipals/Figure/src/ec/edu/espe/Figure/model/Circle.java
package ec.edu.espe.Figure.model;

/**
 *
 * @author Karla Ansatuña
 */
public class Circle extends Shape{
    
    private float radius; 
    public void calculateArea(float radius) {
        double area = Math.PI * Math.pow(radius,2);
        System.out.println("The area of the circle is >"+area);
    }
    public float getRadius() {
        return radius;
    }

    public void setRadius(float radius) {
        this.radius = radius;
    }

    public Circle(float radius) {
        this.radius = radius;
    }

    @Override
    public String toString() {
        return "Circle{" + "radius=" + radius + '}';
    }
    public void calculatePerimeter(float radius) {
        double perimeter = 2*Math.PI*radius;
        System.out.println("The perimeter of teh circle is >"+perimeter);
    }



    @Override
    public void calculateArea() {
    }

    @Override
    public void calculatePerimeter() {
    }


}
